lib/OpenQA/WebAPI/Controller/API/V1/JobGroup.pm

```diff
@@ -344,4 +326,45 @@ sub delete {
     $self->render(json => $event_data);
 }
 
+=over 4
+
+=item build_results()
+
+Shows build results for a job group, similar to what the group_overview page
+provides.
+
+Currently it does not support parent job groups.
```

Comment:
Because it is unclear to me, will the user get an error message? For sake's simplicity, I guess, making it explicit is better in this case.

Reply:
The parent job groups are in a different table and the ids are overlapping with job group ids.
For the WebUI we have `/parent_group_overview/x` and `/group_overview/x`, so we would need also a new route for parent groups, if people want to have that.

Reply:
Got it, thanks!